### PR TITLE
Fix UI Alignment for copy to clipboard

### LINKED ISSUE #611: [UI] Alignment for copy to clipboard
**Current State of the Meshery.io site**
Alignment issue for copy to clipboard
bug

**Desired State of the Meshery.io site**
This should be properly aligned.

---
### Contributor Resources

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by sidecodes**
Signed-off-by: Mohd Talha <mohmmad.talha@outlook.com>

**Description**

This PR fixes #611

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.
 

ui-alignment

### CODE REVIEW

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
@sidecodes a better approach would be to increase the z-index, this is not looking that good

**Comment by sidecodes** [comment]
> Thank you@sidecodes for the PR, a better approach would be to increase the z-index, this is not looking that good

Sure I will change it as requested.

**Comment by warunicorn19** [comment]
Thank you sir.

File: _includes/getting-started.html [hunk 1]
```diff
@@ -99,7 +99,7 @@ <h3>Bash using Mac or Linux</h3>
                   <a class="btn tooltip" data-clipboard-target="#start-k8s" data-clipboard-text="curl -L https://git.io/meshery | PLATFORM=kubernetes bash -"
                     onmouseout="resetCopyText(this)" style="position:absolute; top: 0.10rem; right: 0.5rem; font-size: 22px;">
                     <i class="far fa-copy"></i>
-                    <span class="tooltiptext" style="font-size: 15px; width: 140px; height: 40px; padding: 0; line-height: 40px; top: 3rem; left: -20px;">Copy to clipboard</span>
+                    <span class="tooltiptext" style="font-size: 15px; width: 140px; height: 40px; padding: 0; line-height: 40px; top: 2.5rem; left: -70px;">Copy to clipboard</span>
```
Comment (warunicorn19) [inline _includes/getting-started.html:102]:
```suggestion
                    <span class="tooltiptext" style="font-size: 15px; width: 140px; height: 40px; padding: 0; line-height: 40px; top: 2rem; left: -80px;">Copy to clipboard</span>
```

Comment (sidecodes) [reply _includes/getting-started.html]: Updated as suggested. Please have a look. @warunicorn19 

updated-ui-alignment

**Comment by warunicorn19** [comment]
@sidecodes you seem to have forgotten to make the changes on a couple😅, like KinD, Helm, etc. could you please take a look and make the tooltip consistent on all the snippets

**Comment by sidecodes** [comment]
> 
> 
> @sidecodes you seem to have forgotten to make the changes on a couple😅, like KinD, Helm, etc. could you please take a look and make the tooltip consistent on all the snippets, they should be in the same position in all.

Changed it for consistency. @warunicorn19  
updated-ui-alignment-2

**Review by warunicorn19 — APPROVED** [review]
Looking good! thank you @sidecodes for this to the end.😄

**Pull request merged**